scripts/analysis.py

```diff
@@ -866,48 +837,48 @@ def capacity_calc(governments, timestep, entry, exit_step):
 
     Returns
     -------
-    tuple
-        (power_dict, num_dict) which holds timeseries
-        of capacity and number of reactors
-        with country_government as key
+    power_dict: dictionary
+        dictionary with key: government,
+        value: timesereies capacity
+    num_dict: dictionary
+        dictionary with key: government,
+        value: timesereis number of reactors
     """
-
     power_dict = collections.OrderedDict()
     num_dict = collections.OrderedDict()
-
     for gov in governments:
         capacity = []
         num_reactors = []
         cap = 0
         count = 0
-        gov_name = gov[0]
-        print(gov)
         for t in timestep:
             for enter in entry:
                 entertime = enter[3]
                 parentgov = enter[2]
                 gov_agentid = gov[1]
                 power_cap = enter[0]
                 if entertime == t and parentgov == gov_agentid:
-                    cap += power_cap/1000
+                    cap += power_cap / 1000
                     count += 1
             for dec in exit_step:
                 exittime = dec[3]
                 parentgov = dec[2]
                 gov_agentid = gov[1]
                 power_cap = dec[0]
                 if exittime == t and parentgov == gov_agentid:
-                    cap -= power_cap/1000
+                    cap -= power_cap / 1000
                     count -= 1
             capacity.append(cap)
             num_reactors.append(count)
-
         power_dict[gov[0]] = np.asarray(capacity)
         num_dict[gov[0]] = np.asarray(num_reactors)
 
     return power_dict, num_dict
 
 
+"""PLOTTER"""
```

Comment:
If this needs so much splitting up, maybe these pieces deserve to be in a more structured set of  files...